beamercolorthemelmu.sty

```diff
@@ -3,21 +3,29 @@
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% LMU Colors %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
 % LMU Corporate Design Manual
-\definecolor{lmu@green}{RGB}{0,148,64}
-%\definecolor{lmu@darkgray}{RGB}{127,127,127}
-%\definecolor{lmu@darkgray}{RGB}{96,96,96}
-\definecolor{lmu@darkgray}{HTML}{23373b}
-\definecolor{lmu@lightgray}{RGB}{127,127,127}
+% https://www.lmu.de/de/die-lmu/struktur/zentrale-universitaetsverwaltung/kommunikation-und-presse/lmu-brand-guide/designgrundsaetze/farben/index.html
+\definecolor{lmu@green}{RGB}{0,136,58}
+\definecolor{lmu@black}{RGB}{35,35,35}
+\definecolor{lmu@white}{RGB}{255,255,255}
+
+\definecolor{lmu@darkgray}{RGB}{98,100,104}
+\definecolor{lmu@mediumgray}{RGB}{192,193,195}
+\definecolor{lmu@brightgray}{RGB}{230,230,231}
+\definecolor{lmu@lightgray}{RGB}{245,245,245}
+
+\definecolor{lmu@blue}{RGB}{15,25,135}
+\definecolor{lmu@cyan}{RGB}{0,159,227}
+\definecolor{lmu@violet}{RGB}{140,64,145}
+\definecolor{lmu@red}{RGB}{215,25,25}
+\definecolor{lmu@orange}{RGB}{241,135,0}
 
 % Webstyleguide
-%\definecolor{lmu@background}{RGB}{239,239,233}
-%\definecolor{lmu@background1}{RGB}{206,206,194}
 \definecolor{lmu@background}{RGB}{255,255,255}
 \definecolor{lmu@background1}{RGB}{255,255,255}
 
-\definecolor{lmu@background2}{RGB}{220,220,211}
-\definecolor{lmu@normaltext}{RGB}{0,108,48}
-\definecolor{lmu@visitedhyperlink}{RGB}{102,136,102}
+% normal text darkgray
+\definecolor{lmu@normaltext}{RGB}{98,100,104}
```

Comment:
The contrast of the normal text is really low that way. On the website they use `lmu@black` but in the pptx it's `lmu@darkgray`... should we follow the LMU's own advice and use proper black?

> Das setzt beispielsweise voraus, dass der Hell-Dunkel-Kontrast der Typografie hoch genug ist und solche Kombinationen von Vorder- und Hintergrundfarbe vermieden werden, die Menschen mit Beeinträchtigung (wie Farbenblindheit) ausschließen

```suggestion
\definecolor{lmu@normaltext}{RGB}{35,35,35}
```